packages/backend/src/api/controllers/OfferController.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  OfferRecord,
+  OfferRepository,
+} from '../../peripherals/database/OfferRepository'
+
+export class OfferController {
+  constructor(private offerRepository: OfferRepository) {}
+
+  async postOffer(offer: OfferRecord) {
+    await this.offerRepository.add([offer])
```

Comment:
We want some validation here that the offer actually makes sense given the user's position